_partials/bpm-suite-starter-kit.html.slim

```diff
@@ -0,0 +1,121 @@
+.row
+  .large-24.columns
+    p
+      | The purpose of this kit is to provide the means to get started with the
```

Comment:
These longer sections of prose would be much better as asciidoc. See http://www.rubydoc.info/gems/slim/frames#Embedded_engines__Markdown______ for how to use the embedded engines.